apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/GrpcRdaSource.java

```diff
@@ -262,7 +266,18 @@ public void close() throws Exception {
         channel.shutdown();
       }
       if (!channel.isTerminated()) {
-        channel.awaitTermination(5, TimeUnit.SECONDS);
+        try {
+          channel.awaitTermination(1, TimeUnit.MINUTES);
+        } catch (InterruptedException ex) {
+          LOGGER.info("caught InterruptedException while closing ManagedChannel - retrying once");
+          try {
+            channel.awaitTermination(1, TimeUnit.MINUTES);
+          } catch (InterruptedException ex2) {
+            LOGGER.info(
+                "caught second InterruptedException while closing ManagedChannel - calling shutdownNow");
```

Comment:
A warn could unecessarily trigger an alarm and this situation is harmless.  I'm just noting it in the log so we can look for it in case it happens often.